features-json/font-variant-ligatures.json

```diff
@@ -0,0 +1,340 @@
+      "59":"u",
+      "60":"u",
+      "61":"u",
+      "62":"u",
+      "63":"u",
+      "64":"u",
+      "65":"u",
+      "66":"u",
+      "67":"u",
+      "68":"u",
+      "69":"y",
+      "70":"y",
+      "71":"y",
+      "72":"y",
+      "73":"y"
+    },
+    "safari":{
+      "3.1":"u",
+      "3.2":"u",
+      "4":"u",
+      "5":"u",
+      "5.1":"u",
+      "6":"u",
+      "6.1":"u",
+      "7":"u",
+      "7.1":"u",
+      "8":"u",
+      "9":"u",
+      "9.1":"u",
+      "10":"u",
+      "10.1":"u",
+      "11":"u",
+      "11.1":"u",
+      "12":"y",
+      "TP":"y"
+    },
+    "opera":{
+      "9":"n",
+      "9.5-9.6":"n",
+      "10.0-10.1":"n",
+      "10.5":"n",
+      "10.6":"n",
+      "11":"n",
+      "11.1":"n",
+      "11.5":"n",
+      "11.6":"n",
+      "12":"n",
+      "12.1":"n",
+      "15":"u",
+      "16":"u",
+      "17":"u",
+      "18":"u",
+      "19":"u",
+      "20":"u",
+      "21":"u",
+      "22":"u",
+      "23":"u",
+      "24":"u",
+      "25":"u",
+      "26":"u",
+      "27":"u",
+      "28":"u",
+      "29":"u",
+      "30":"u",
+      "31":"u",
+      "32":"u",
+      "33":"u",
+      "34":"u",
+      "35":"u",
+      "36":"u",
+      "37":"u",
+      "38":"u",
+      "39":"u",
+      "40":"u",
+      "41":"u",
+      "42":"u",
+      "43":"u",
+      "44":"u",
+      "45":"u",
+      "46":"u",
+      "47":"u",
+      "48":"u",
+      "49":"u",
+      "50":"u",
+      "51":"u",
+      "52":"u",
+      "53":"u",
+      "54":"u",
+      "55":"u",
+      "56":"y"
+    },
+    "ios_saf":{
+      "3.2":"n",
+      "4.0-4.1":"u",
+      "4.2-4.3":"u",
+      "5.0-5.1":"u",
+      "6.0-6.1":"u",
+      "7.0-7.1":"n",
+      "8":"n",
+      "8.1-8.4":"n",
+      "9.0-9.2":"n",
+      "9.3":"u",
+      "10.0-10.2":"u",
+      "10.3":"u",
+      "11.0-11.2":"y",
+      "11.3-11.4":"y",
+      "12":"y"
+    },
+    "op_mini":{
+      "all":"n"
+    },
+    "android":{
+      "2.1":"n",
+      "2.2":"n",
+      "2.3":"n",
+      "3":"n",
+      "4":"n",
+      "4.1":"n",
+      "4.2-4.3":"n",
+      "4.4":"u",
+      "4.4.3-4.4.4":"u",
+      "67":"u"
+    },
+    "bb":{
+      "7":"n",
+      "10":"u"
+    },
+    "op_mob":{
+      "10":"n",
+      "11":"n",
+      "11.1":"n",
+      "11.5":"n",
+      "12":"n",
+      "12.1":"n",
+      "46":"u"
+    },
+    "and_chr":{
+      "69":"u"
+    },
+    "and_ff":{
+      "62":"u"
+    },
+    "ie_mob":{
+      "10":"n",
+      "11":"n"
+    },
+    "and_uc":{
+      "11.8":"u"
+    },
+    "samsung":{
+      "4":"u",
+      "5":"u",
+      "6.2":"u",
+      "7.2":"u"
+    },
+    "and_qq":{
+      "1.2":"u"
+    },
+    "baidu":{
+      "7.12":"u"
+    }
+  },
+  "notes":"",
+  "notes_by_num":{
+    "1":"First note..."
+  },
+  "usage_perc_y":0,
+  "usage_perc_a":0,
+  "ucprefix":false,
+  "parent":"",
+  "keywords":"font-feature,font-feature-settings,font-variant",
+  "ie_id":"",
+  "chrome_id":"",
+  "firefox_id":"",
+  "webkit_id":"",
+  "shown":true
```

Comment:
Too many unknown support. Please change this line to “shown”:false

Reply:
@HM100 Is it the previous version support, or number of different browsers that's an issue? Just wondering how I can help this along as I'd really like to get more of the font features included.

Reply:
The number of previous versions support is almost of unknown support excluding IE and Edge

Reply:
I've updated as requested

Reply:
I found a bunch more detail on when the feature shipped in almost all of the browsers, so updated with that info and set back to shown: true (please let me know if that's a problem and I'll update again)

Reply:
and please fix those `x` to `y` or `y x` if those versions with `x` are prefixed supported

Reply:
@jpamental Any ETA on fixing the issues?